src/main/java/frc/robot/commands/autos/AutoScoreTwo.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.commands.autos;
+
+import java.util.HashMap;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.commands.FollowPathWithEvents;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj.DriverStation;
+import frc.robot.commands.IntakeSequence;
+import frc.robot.commands.autos.components.AutoScoreOne;
+import frc.robot.subsystems.Lights;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.drive.DriveCal;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.grabber.Grabber;
+
+public class AutoScoreTwo extends SequentialCommandGroup {
+    private PathPlannerTrajectory trajNewGamePiece =
+      PathPlanner.loadPath(
+          "TwoNoBump",
+          new PathConstraints(
+              DriveCal.MAX_LINEAR_SPEED_METERS_PER_SEC,
+              DriveCal.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+
+    private HashMap<String, Command> eventMap = new HashMap<>();
+
+    public AutoScoreTwo(boolean red, boolean fast, DriveSubsystem drive, Arm arm, Grabber grabber, Lights lights) {
+
+    if (red) {
+        trajNewGamePiece =
+            PathPlanner.loadPath(
+                "ScoringLocToChargeStationRed",
+                new PathConstraints(
+                    DriveCal.MAX_LINEAR_SPEED_METERS_PER_SEC,
+                    DriveCal.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+  
+        trajNewGamePiece =
+            PathPlannerTrajectory.transformTrajectoryForAlliance(
+                trajNewGamePiece, DriverStation.Alliance.Red);
+      }
+
+    addRequirements(drive, arm, grabber, lights);
+    
+    eventMap.put("intakeGamePiece", IntakeSequence.interruptibleIntakeSequence(arm, grabber, lights));
```

Comment:
Do we want to do anything special when we're on the way back? What happens if we fail to intake or something, where will the arm be

Reply:
i have added a timeout to the intake sequence with a finallyDo of move back to starting and stop spinning grabber motors

Reply:
I don't think you need the andThen, since the interruptible will do that stuff once it ends (due to cancellation or finishing)